android/app/src/main/java/com/cBreez/client/BreezNotificationService.kt

```diff
@@ -16,91 +26,182 @@ import kotlinx.coroutines.launch
 import org.tinylog.kotlin.Logger
 
 class BreezNotificationService {
+
```

Comment:
`BreezNotificationService` name implies Android service.... perhaps we can rename it to `BreezNotificationHelper` ?

Reply:
Done f57249306cf4e1180de1a060284a73716d50519c